src/main/java/taintedmagic/common/items/equipment/ItemVoidwalkerBoots.java

```diff
@@ -137,9 +137,9 @@ public void onArmorTick (final World world, final EntityPlayer player, final Ite
 
             // speed boost
             if (player.onGround || player.capabilities.isFlying) {
-                float bonus = 0.12F;
+                float bonus = 0.2F;
```

Comment:
Commit also buffs the speed since it was decreased drastically and now it is barely visible and loses to it's counterparts.